docs/zms_svctoken.md

```diff
@@ -0,0 +1,60 @@
+* [Overview](#overview)
+* [Getting Software](#getting-software)
+* [Prerequisites](#prerequisites)
+* [Usage](#usage)
+
+
+## Overview
+----------
+
+ZMS Service Token Client utility generates service tokens
+based on given private key and service details.
+
+## Getting Software
+-------------------
+
+Download latest ZMS Service Token Client utility binary release from Bintray - click
+on the `Files` tab, choose the latest version directory and then
+download the `athenz-utils-<latest-version>-bin.tar.gz` file::
+
+[ ![Download](https://api.bintray.com/packages/yahoo/maven/athenz-utils/images/download.svg) ](https://bintray.com/yahoo/maven/athenz-utils/_latestVersion)
```

Comment:
Noticed that all the utilities are part of the same package in bintray - athenz-utils.
I copied the "Getting Software" and "Preqreuisites" parts from the ZMS Util guide to all relevant docs.